admin/admin.go

```diff
@@ -37,24 +37,29 @@ func New(base duoapi.DuoApi) *Client {
 
 // User models a single user.
 type User struct {
-	Alias1            *string `url:"alias1"`
-	Alias2            *string `url:"alias2"`
-	Alias3            *string `url:"alias3"`
-	Alias4            *string `url:"alias4"`
-	Created           uint64
-	Email             string  `url:"email"`
-	FirstName         *string `url:"firstname"`
-	Groups            []Group
-	LastDirectorySync *uint64 `json:"last_directory_sync"`
-	LastLogin         *uint64 `json:"last_login"`
-	LastName          *string `url:"lastname"`
-	Notes             string  `url:"notes"`
-	Phones            []Phone
-	RealName          *string `url:"realname"`
-	Status            string  `url:"status"`
-	Tokens            []Token
-	UserID            string `json:"user_id"`
-	Username          string `url:"username"`
+	Alias1            *string         `json:"alias1" url:"alias1"`
+	Alias2            *string         `json:"alias2" url:"alias2"`
+	Alias3            *string         `json:"alias3" url:"alias3"`
+	Alias4            *string         `json:"alias4" url:"alias4"`
+	Created           uint64          `json:"created"`
+	DesktopTokens     []interface{}   `json:"desktop_tokens"`
```

Comment:
This `DesktopTokens` field can be completely removed. It was inadvertently added to the documentation.